cluster/mesos/docker/docker-compose.yml

```diff
@@ -157,10 +157,12 @@ scheduler:
   volumes:
   - ./static-pod.json:/opt/static-pods/static-pod.json
 keygen:
-  image: mesosphere/kubernetes-mesos-keygen
+  image: mesosphere/kubernetes-keygen:v1.0.0
```

Comment:
Believe me, a curl is a blocker. You can easily use the cluster offline in
a train. This curl breaks it.